service/src/main/java/bio/terra/cbas/controllers/MethodsApiController.java

```diff
@@ -216,14 +240,19 @@ public ResponseEntity<MethodListResponse> getMethods(
 
     if (methodVersionId != null) {
       methodDetails =
-          List.of(methodVersionToMethodDetails(methodVersionDao.getMethodVersion(methodVersionId)));
+          List.of(
+              methodVersionToMethodDetails(
+                  methodVersionDao.getMethodVersion(methodVersionId), showMethodDetails));
     } else {
       List<Method> methods =
           methodId == null ? methodDao.getMethods() : List.of(methodDao.getMethod(methodId));
       boolean nullSafeShowVersions = showVersions == null || showVersions;
+      boolean nullSafeShowMethodDetails = showMethodDetails == null || showMethodDetails;
```

Comment:
this is more of a "note to self, look into this" than feedback or a change request, because I see it's a pattern that you didn't introduce -- but it's not obvious how these variables might end up being `null`. Does it happen because it gets its arguments based on the openapi spec? I wonder...